planner/core/integration_test.go

```diff
@@ -293,14 +292,7 @@ func (s *testIntegrationSerialSuite) TestNoneAccessPathsFoundByIsolationRead(c *
 
 	_, err = tk.Exec("select * from t")
 	c.Assert(err, NotNil)
-	c.Assert(err.Error(), Equals, "[planner:1815]Internal : Can not find access path matching 'tidb_isolation_read_engines'(value: 'tiflash') and tidb-server config isolation-read(engines: '[tikv tiflash tidb]'). Available values are 'tikv'.")
```

Comment:
It's better to keep this test. It shows how the session variable and server config value works together when they are conflicted.

Reply:
It's a useless test, global config only init the session. Even if we keep the test, change the config, it infects nothing.

Reply:
Yeah, we need to test that it "affects nothing". Make sure once the behavior  is changed mistakenly in the future, this test can fail.